### DISCUSSION TITLE
Same video in two lessons updraft.cyfrin.io service

### DISCUSSION THREAD

**Discussion by veebull** [Cyfrin/security-and-auditing-full-course-s23#92] [General]
In the course there is same video in two lessons on updraft.cyfrin.io.
**Section 3: Your First Audit | PasswordStore**
`Lesson 8: Recon: Understanding the code`
is equal to
`Lesson 9: Exploit: Access control`

It is maybe bug maybe feature. I don't know but for me pretty strange.

**Comment by cromewar** [comment]
Hello @armbull, thanks for reporting this issue.
Endded is a bug, and we just updated the section with the actual video and the changes will reflect in some minutes, thanks a lot.
